Firestore/Swift/README.md

```diff
@@ -0,0 +1,38 @@
+# Firestore Swift Extensions
+
+This directory contains source files for Firestore's Swift extensions SDK.
+
+## Installation
+
+The FirebaseFirestoreSwift module is currently only available for installation
+through CocoaPods. To add FirebaseFirestoreSwift to your project, add the
+following to your Podfile
+
+```ruby
+pod 'FirebaseFirestoreSwift'
+```
+
+You will also need to import the FirebaseFirestoreSwift module in any source
+file that depends on it, since it's not automatically bundled with the
+Firestore module.
+
+```swift
+import Firebase
+import FirebaseFirestoreSwift
+```
+
+The FirebaseFirestoreSwift module does not provide any additional utility to
+Objective-C projects, and is thus not recommended for non-Swift projects.
```

Comment:
"thus" is about the how of something and "therefore" is more about why--expressing a causal relationship (you mentally replace it with "because of this"). I think therefore is preferred here.

Also consider keeping the "is not" together.

Reply:
Done.